Src/FluentAssertions/Formatting/EnumerableValueFormatter.cs

```diff
@@ -7,14 +7,16 @@ namespace FluentAssertions.Formatting
 {
     public class EnumerableValueFormatter : IValueFormatter
     {
+        protected virtual int MaxItems { get; } = 32;
```

Comment:
Why do you need this?

Reply:
To limit/extend the number of formatted items for a custom formatter.
Am I just repeating what the code does, instead of why it's necessary?

Reply:
Ah, you made the `EnumerableValueFormatter ` also extensible. I didn't get that immediatelly.